docs/source/docs/index.rst

```diff
@@ -139,7 +139,7 @@ Contents
    :maxdepth: 1
    :caption: Cutting Cloud Costs
 
-   ../examples/spot-jobs
+   ../examples/managed-jobs
```

Comment:
Also for brainstorming: this may be more indirect than before, where people could clearly see "ah managed spot is clearly something I can use to save costs".